README.md

```diff
@@ -87,4 +92,5 @@ The PR process is managed by github workflows, where the code undergoes several
 The list of mainnet addresses for different chains and their full contract configuration can be found in the following locations:
 - [Governance and Token addresses](https://github.com/valory-xyz/autonolas-governance/blob/main/docs/configuration.json);
 - [Agent Services Functionality addresses](https://github.com/valory-xyz/autonolas-registries/blob/main/docs/configuration.json);
-- [Tokenomics addresses](https://github.com/valory-xyz/autonolas-tokenomics/blob/main/docs/configuration.json).
+- [Tokenomics addresses](https://github.com/valory-xyz/autonolas-tokenomics/blob/main/docs/configuration.json);
+- [Lockbox Solana addresses](https://github.com/valory-xyz/lockbox-solana/blob/docs/configuration.json).
```

Comment:
I suggest to add this link in the future since now is broken

Reply:
Let's leave this PR unmerged before we deploy the lockbox :)

Reply:
ok, cool!